src/vm/exceptionhandling.cpp

```diff
@@ -1814,7 +1814,25 @@ CLRUnwindStatus ExceptionTracker::ProcessOSExceptionNotification(
 
         if (fIsFrameLess)
         {
-            pGSCookie = (GSCookie*)cfThisFrame.GetCodeManager()->GetGSCookieAddr(cfThisFrame.pRD,
+            PREGDISPLAY pRD = NULL;
```

Comment:
@janvorli Could you please take a look for #8889, #8914, #8964, #8993? The changes in ``REGDISPLAY`` will make a conflict with these PRs, and thus I would like to revise ``REGDISPLAY``  after merging them.